xml/System.Diagnostics/ActivityCreationOptions`1.xml

```diff
@@ -59,8 +61,8 @@
         <ReturnType>System.Collections.Generic.IEnumerable&lt;System.Diagnostics.ActivityLink&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the enumeration of activity links associated with the activity.</summary>
```

Comment:
>Gets the enumeration of activity links associated with the activity [](start = 17, length = 67)

Get the ebumeration of the activity links that the Activity will be created with.

Reply:
```suggestion
        <summary>Gets the enumeration of activity links that the activity will be created with.</summary>
```